rock.osdeps

```diff
@@ -235,7 +235,7 @@ dc1394:
 
 vtk:
     debian,ubuntu:
-        '14.04,14.10,15.04,15.10':
+        '14.04,14.10,15.04,15.10,16.04':
```

Comment:
when changing the osdep to 'default: [libvtk5-qt4-dev,libvtk5-dev]' and keeping the use of libvtk6-dev in rock osdeps, it conflicts with the installation of python-vtk6.
I purged all libvtk* before